layouts/partials/survey-notice.html

```diff
@@ -0,0 +1,11 @@
+{{ if .Param "survey-notice"}}
+  <section id="survey-notice">
+    <main>
+      <div class="content survey-notice">
+        <h3>
+          The SIG Docs team needs your feedback! Please take a <a href="https://www.surveymonkey.com/r/8R237FN" target="_blank">short survey</a> so we can improve the Kubernetes documentation.
```

Comment:
People reading this might click the link before finding that it's a survey about the docs rather than about feedback.
(“SIG Docs” should make that clear, but it might not).

If that's a concern, maybe:
```suggestion
          Tell us your views on this documentation. Please take a short <a href="https://www.surveymonkey.com/r/8R237FN" target="_blank">survey</a>&mdash;Kubernetes SIG Docs wants your feedback!
```

Reply:
Thanks for the suggestions!